apps/storefront/app/grunnleggende/for-utviklere/storrelsar/page.mdx

```diff
@@ -0,0 +1,65 @@
+import { CodeIcon } from '@navikt/aksel-icons';
+
+import { MenuPageLayout } from '@layouts';
+import { CodeSnippet } from '@repo/components';
+
+export const metadata = {
+  title: 'Størrelsar',
+  description: 'Informasjon om korleis du bruker størrelsar på frontend.',
+};
+
+export default ({ children }) => (
+  <MenuPageLayout
+    content={children}
+    data={{
+      title: 'Størrelsar',
+      date: '2. januar 2025',
+      icon: <CodeIcon />,
+      color: 'blue',
+    }}
+  />
+);
+
+HTML data-attributt vert brukt for å definere størrelse på ein komponent. Komponentane kan nytte alle alle størrelsar.
+
+Våre `size`-variablar (`--ds-size-*`) endrar seg basert på den lokale skriftstørrelsen. Du kan lett endre den ved bruk av `data-size`.
```

Comment:
Skrevet om til noe slikt? Kom med bare forslag så du kan skrive det riktig nynorsk 😅 

> Våre `size`-variablar (`--ds-size-*`) endrar seg basert på den lokale skriftstørrelsen (formel basert på `em`). Du kan lett endre den ved bruk av `data-size` for våre anbefalte størrelser eller definer din egen størrelse ved å sette `font-size`.

Reply:
Våre size-variablar (--ds-size-*) endrar seg basert på den lokale skriftstørrelsen (formel basert på `em`). Du kan lett endre den ved bruk av `data-size` for våre anbefalte størrelsar, eller definere din eigen størrelse ved å sette `font-size` i CSS.

^ Funka dette?

Reply:
Ser bra ut den :)